src/components/flyout/deviceTagFlyout.js

```diff
@@ -124,13 +132,40 @@ class DeviceTagFlyout extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    if (!_.isEqual(nextProps.devices, this.props.devices)) {
-      this.computeState(nextProps, 'componentWillReceiveProps');
+    const { devices, twinUpdateJobs } = nextProps;
+    if (!_.isEqual(devices, this.props.devices)) {
+      this.setState({
```

Comment:
For adding parameters on multiple lines, follow either all the parameters on one line, or each parameters has its own line.
```
this.setState(
   { jobApplied: false },
   () => this.computeState(nextProps)
);
```